src/esp/bindings/SensorBindings.cpp

```diff
@@ -37,6 +38,94 @@ esp::scene::SceneNode* nodeGetter(T& self) {
 namespace esp {
 namespace sensor {
 
+// Get the torch tensor or numpy array buffer of the Sensor, initialize if it
+// does not exist yet
+template <class T>
+
+auto buffer(T& self, int gpuDevice) {
+  py::handle handle = py::cast(self);
+  if (!py::hasattr(handle, "__buffer")) {
+    if (self.specification()->gpu2gpuTransfer) {
+#ifdef ESP_BUILD_WITH_CUDA
+      auto torch = py::module_::import("torch");
```

Comment:
Torch isn't a hard dependency even with CUDA enabled. Not sure if throws a graceful error, can you check?

Reply:
Thanks @Skylion007 , it builds without errors even without the #ifdef